src/main/scala/com/cloudant/clouseau/IndexService.scala

```diff
@@ -117,6 +121,11 @@ class IndexService(ctx: ServiceContext[IndexServiceArgs]) extends Service(ctx) w
         pendingSeq = newSeq
         logger.debug("Pending sequence is now %d".format(newSeq))
         'ok
+      case SetPurgeSeqMsg(newSeq: Long) =>
+        pendingPurgeSeq = newSeq
+        logger.debug("Pending purge sequence is now %d".format(newSeq))
+        commit(pendingSeq, pendingPurgeSeq)
```

Comment:
Hi @rnewson .
I triggered commit when the purgeSeq is updated. Will it cost too much?